storage/tianmu/system/cacheable_item.cpp

```diff
@@ -23,153 +23,172 @@
 
 namespace Tianmu {
 namespace system {
-CacheableItem::CacheableItem(char const *owner_name, char const *object_id, int _default_block_size) {
-  default_block_size = _default_block_size;
+CacheableItem::CacheableItem(char const *owner_name, char const *object_id, int default_block_size) {
+  default_block_size_ = default_block_size;
+  max_file_id_ = 0;
```

Comment:
These class variables can be inited during definition in class. Follow c++11 std.
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules